ebl/v3/surrender/EBL_SUR_v3.0.0.yaml

```diff
@@ -278,9 +280,15 @@ components:
         endorsementChain:
           type: array
           description: |
-            The endorsement chain consist of (a number of) endorsement actions that were performed **after** the eBL bill was issued.
+           A list of one or more endorsement related actions that were performed on or after the issuance of the eBL. It is equivalent to the "back side" of the physical bill of lading. The type of actions recorded in the endorsement chain as defined by the DCSA standard are listed below:
+
+            - **Issue:** The act of issuing an eBL i.e. making the eBL available to the receiver.
+            - **Endorse:** The act of transferring the rights and obligations associated with the eBL to a specific named party, allowing them to claim or deal with the goods. The user in control of the eBL may endorse the eBL in their turn to another named party. Only applicable to To-Order eBL (`isToOrder=true`).
+            - **Sign:** A general-purpose signature that can be used by any party to mark their possession of the eBL. Similar to how any possessor in the physical world can put a physical signature on the paper bill of lading. The endorsement chain as defined by DCSA does not record any transfer of possession of the eBL, unless a signature is added to it.
+            - **Request surrender for amendment:** The presentation (by transfer) of the eBL to the Issuer, or another user appointed by the Issuer, by a user entitled to do so for the puprpose of amending the eBL.
+            - **Request surrender for delivery:** The presentation (by transfer) of the eBL to the Issuer, or another user appointed by the Issuer, by a user entitled to do so for the purpose of claiming delivery of the goods.
 
-            For straight B/Ls, the endorsement chain can be omitted. For negotiable eBLs, the endorsement chain must be present, as the carrier receiving the surrender request and will check all endorsements before accepting the eBL. The endorsement chain is the electronic version of the "back side" of the paper world B/L.
+            **Note:** DCSA member carriers have agreed that the name "endorsement chain" is still the correct name for this list of actions.
```

Comment:
Should the endorsement chain be highlighted?

Reply:
Good suggestion